components/inputs/test/input-number.test.js

```diff
@@ -58,90 +56,145 @@ describe.only('d2l-input-number', () => {
 			});
 		});
 
-		['autocomplete', 'max', 'maxFractionDigits', 'min', 'minFractionDigits', 'placeholder', 'title', 'value'].forEach((name) => {
+		['autocomplete', 'max', 'min', 'placeholder', 'title', 'value'].forEach((name) => {
 			it(`should default "${name}" property to 'undefined' when unset`, async() => {
-				expect(elem.value).to.equal(undefined);
+				expect(elem[name]).to.equal(undefined);
 			});
 		});
 
 		it('should default "formattedValue" property to empty when unset', () => {
 			expect(elem._formattedValue).to.equal('');
 		});
+
 	});
 
 	describe('min and max fraction digits', () => {
-		[
-			{
-				name: 'should round down to the nearest integer',
-				fixture: integerFixture,
-				value: 15.2345,
-				expectedValue: 15,
-				expectInnerValue: '15'
-			},
-			{
-				name: 'should round up to the nearest integer',
-				fixture: integerFixture,
-				value: 15.6345,
-				expectedValue: 16,
-				expectInnerValue: '16'
-			},
-			{
-				name: 'should automatically add zeroes to match min fraction digits',
-				fixture: minMaxFractionDigitsFixture,
-				value: 1,
-				expectedValue: 1,
-				expectInnerValue: '1.00'
-			},
-			{
-				name: 'should automatically round up to match max fraction digits',
-				fixture: minMaxFractionDigitsFixture,
-				value: 1.2345,
-				expectedValue: 1.235,
-				expectInnerValue: '1.235'
-			},
-			{
-				name: 'should automatically round down to match max fraction digits',
-				fixture: minMaxFractionDigitsFixture,
-				value: 1.2344,
-				expectedValue: 1.234,
-				expectInnerValue: '1.234'
-			}
-		].forEach((test) => {
-			it(test.name, async() => {
-				const elem = await fixture(test.fixture);
-				elem.value = test.value;
-				await elem.updateComplete;
-				expect(elem.value).to.equal(test.expectedValue);
-				expect(getInnerInputValue(elem)).to.equal(test.expectInnerValue);
-			});
```

Comment:
The way these were working (by setting the value _after_ initialization) was what caused us to not notice the problems. So kept all the tests but just rewrote them a different way.